src/libraries/System.Private.CoreLib/src/System/Threading/NativeRuntimeEventSource.PortableThreadPool.NativeSinks.cs

```diff
@@ -248,5 +250,20 @@ public unsafe void ThreadPoolIOPack(NativeOverlapped* nativeOverlapped)
         {
             LogThreadPoolIOPack(NativeOverlapped, Overlapped, ClrInstanceID);
         }
+
+
+        [Event(59, Level = EventLevel.Informational, Message = Messages.MinMaxThreads, Task = Tasks.ThreadPoolMinMaxThreads, Opcode = EventOpcode.Info, Version = 0, Keywords = Keywords.ThreadingKeyword)]
+        public unsafe void ThreadPoolMinMaxThreads(
+            short MinWorkerThreads,
+            short MaxWorkerThreads,
+            short MinIOCompletionThreads,
+            short MaxIOCompletionThreads,
```

Comment:
The data type of these arguments should match the data type of the values in `ClrEtwAll.man` since the manifest may also be generated from methods attributed with `EventAttribute`, so if the data type is changed, this would have to be changed too. Similarly in the other NRES files.